app/models/copy_staging/restore_work.rb

```diff
@@ -32,27 +32,24 @@ module CopyStaging
   # could be disastrous. This object can take thread_pool_size as an init
   # argument, but the wrapping rake task isn't currently written to exersize that.
   class RestoreWork
-    REMOTE_STORE_STORAGE_KEY = :remote_store_storage
-    REMOTE_DERIVATIVES_STORAGE_KEY = :remote_derivatives_storage
+    ORIGINALS_STORAGE   = [:store, :video_store]
+    DERIVATIVES_STORAGE = [:kithe_derivatives, :restricted_kithe_derivatives]
 
     attr_accessor :json_file, :thread_pool, :tracked_futures, :thread_pool_size
 
     # @param json_file path to file of json of the format we expect, normally
     #   exported from correspoinding SerializeWork file.
     #
     # @param thread_pool_size how big a thread pool to use for our parallel
-    #   bytestrea copy operations. See docs at top of class. Defaults to 50,
+    #   bytestream copy operations. See docs at top of class. Defaults to 50,
     #   could be faster if we made it unlimited, but could also be more disastrous.
     def initialize(json_file:, thread_pool_size: 50)
       @json_file = json_file
       @thread_pool_size = thread_pool_size
-
       @thread_pool = Concurrent::FixedThreadPool.new(thread_pool_size)
       @tracked_futures = Concurrent::Array.new
-
-      # make sure they get registered
-      remote_store_storage
-      remote_derivatives_storage
+      @storage_map = Hash[ (ORIGINALS_STORAGE + DERIVATIVES_STORAGE).collect { |v| [v, "remote_#{v.to_s}".to_sym  ] } ]
```

Comment:
I honestly would have just written this out the long way, for clarity... but no big deal, not requiring the change, just saying.